src/bandersnatch_docker_compose/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  bandersnatch:
+    image: pypa/bandersnatch:latest
+    volumes:
+      - "./config:/conf"
+      - "./data:/srv/pypi/web"
+    networks:
+      - bandersnatch_net
+  bandersnatch_nginx:
+    image: bandersnatch_nginx
+    build: .
+    volumes:
+      - "./data:/data/pypi/web:ro"
+    ports:
+      - "40080:80"
+    networks:
+      - bandersnatch_net
+
+networks:
+  bandersnatch_net:
+    driver: bridge
+    driver_opts:
+      com.docker.network.driver.mtu: 800
```

Comment:
800 if very low. How did you calculate that was needed?

I've never had to lower MTU on my docker networks, but I don't really do super heavy networking on any of my docker containers ...

Reply:
Good catch! I took the very low MTU to make sure that it definitely works. 1400 should also do the trick, but I would rather extend the README with hints regarding the numbers.